src/components/Inputs/ZipInput/ZipInput.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { INPUT_MODES, INPUT_PATTERNS } from '../../../constants'
 import { TextInput } from '../TextInput'
 
 export const ZipInput = (props) => {
-  return <TextInput {...props} />
+  return (
+    <TextInput
+      {...props}
+      inputMode={INPUT_MODES.NUMERIC}
+      pattern={INPUT_PATTERNS.NUMERIC}
```

Comment:
So this is probably only needed for envs NOT mobile huh? I'm thinking the inputmode on mobile will give the numeric UI that only allows 0-9 anyhoo huh? Just thinking aloud. Yeah, but we still need for desktop et al huh?